README.md

```diff
@@ -27,7 +27,18 @@ After finish the setup, access this url: http://localhost/.
 ## Start development of apps
 
 You will need create (or clone) the folder of the app that you will work inside the folder `volumes/nextcloud/apps-extra`.
-Good work!
+
+It's not required install all dependencis like php or nodejs to develop apps, with this project is only use the bash in container to compile app.
+
+## Sample using the [LibreSign](https://github.com/LibreSign/libresign)
+
+To install LibreSign in the structure of develop is required [up servicer](#up-services). After nextcloud config and install.
+  - open folder `/volumes/nextcloud/app-extra`
+  - clone project with `git clone https://github.com/LibreSign/libresign.git`
+  - open bash in nextcloud container with `docker compose exec nextcloud /bin/bash`
+  - open folder `/app-extra`
+  - run commands to compile LibreSign, commands like: `make serve`
```

Comment:
```suggestion
  - Now you can run all the necessaries commands to build the project, i.e:
    ```bash
    # download composer dependencies
    composer install
    # download JS dependencies
    npm ci
    # build and watch JS changes
    npm run watch
    ```
```